logic/system.js

```diff
@@ -242,6 +242,24 @@ async function getLndConnectUrls() {
 
 }
 
+async function requestDebug() {
+    try {
+        await diskLogic.writeSignalFile(`debug`);
```

Comment:
```suggestion
        await diskLogic.writeSignalFile('debug');
```

Using ` is not needed here.